docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -1232,11 +1232,11 @@ The interval string is a possibly signed sequence of decimal numbers, followed b
 
 ### ha_listen_address
 
-Listen address/hostname and port to receive unified alerting messages for other Grafana instances. The port is used for both TCP and UDP. It is assumed other Grafana instances are also running on the same port. The default value is `0.0.0.0:9094`.
+Listen IP address and port to receive unified alerting messages for other Grafana instances. The port is used for both TCP and UDP. It is assumed other Grafana instances are also running on the same port. The default value is `0.0.0.0:9094`.
 
 ### ha_advertise_address
 
-Explicit address/hostname and port to advertise other Grafana instances. The port is used for both TCP and UDP.
+Explicit IP address and port to advertise other Grafana instances. The port is used for both TCP and UDP.
```

Comment:
```suggestion
Explicit IP address and port to advertize other Grafana instances. The port is used for both TCP and UDP.
```

Reply:
Just that we should use American English (realize you didn't make this change-but figure we can take advantage :)

Reply:
:) codespell thinks otherwise 
```
docs/sources/setup-grafana/configure-grafana/_index.md:1239: advertize ==> advertise
```
the CI failed